tools/deletemincsqlwrapper.pl

```diff
@@ -69,5 +76,11 @@
     my $minc_delete_log = `$minc_delete_cmd`;
     print $minc_delete_log . "\n";
 
+    if ($insertminc) {
+      my $tar_loader_cmd  = "../uploadNeuroDB/tarchiveLoader -profile " . $profile . " -seriesuid " . $fF->{'SeriesUID'} . " -verbose -globLocation " . $fF->{'ArchiveLocation'};
+      print $tar_loader_cmd . "\n";
```

Comment:
@gluneau 
not sure where the ArchiveLocation is coming from. You need to join on the tarchive table to get this location.
(running this with the `insertminc` option gives the error:
```../uploadNeuroDB/tarchiveLoader -profile prod -seriesuid 1.3.12.2.1107.5.2.43.67007.30000016111018154618200000794 -verbose -globLocation 
...
usage: ../uploadNeuroDB/tarchiveLoader </path/to/DICOM-tarchive> [options]
       ../uploadNeuroDB/tarchiveLoader -help to list options


	ERROR: You must specify a valid tarchive and an existing profile.
```